global.json

```diff
@@ -3,7 +3,7 @@
     "dotnet": "5.0.100-rc.2.20479.15",
     "runtimes": {
       "dotnet": [
-        "$(MicrosoftNETCoreAppInternalPackageVersion)"
+        "$(MicrosoftNETCoreAppRuntimePackageVersion)"
```

Comment:
This is not correct, as it needs to be a non-stable version. It works now because the runtime is non-stable.

Reply:
I take it that's also the case for WPF as well. I'll update both to use a non-shipping package then.